packages/classes/docs/PAYLOAD.md

```diff
@@ -0,0 +1,34 @@
+# Payload
+
+Payloads provide a consistant structured format for TV data. If you are using or contributing to the TV Kitchen you are probably going to be working with Payloads.
+
+For example:
+
+- `Ingestion Engines` produce payloads containing video data.
+- `Appliances` consume payloads, invoke their transformation or other process, and emit payloads with new types of data.
+- `The Countertop` announces new payloads as they are generated.
+- The TV Kitchen stream API emits payloads as they are announced by `The Countertop`. 
+
+## Attributes
+
+Payloads have the following attributes:
+
+- **data** (Buffer): The data contained by the payload.
+- **type** (string): The [type](https://github.com/tvkitchen/base/blob/master/packages/constants/src/dataTypes.js) of Payload.
+- **createdAt** (String): The ISO 8601 timestamp that the payload was created.
+- **timestamp** (String): The ISO 8601 timestamp that specifies when, in absolute terms, the Payload's data starts.
+- **duration** (Number): The number of milliseconds that the Payload's data represents within the stream.
+- **position** (Number): The number of milliseconds into the stream where the Payload's data starts.
+
+For example, if a stream started at 12:00 GMT on March 15th, 2020, a Payload containing 1 second of captions from 2 minutes into the stream might have the following values:
+
+```json
+{
+	"data": {"type":"Buffer","data":[65,32,99,97,112,116,105,111,110,46]},
+	"type": "TEXT.BLOB",
+	"createdAt": "2020-03-15T12:02:00+5015",
+	"timestamp": "2020-03-15T12:02:00+0000",
```

Comment:
I'll just note that these examples don't match the test data, which seems significant somehow. I get that they're both valid but we should probably at least internally formalize around one representation?

Reply:
Test data reflects what Date's toISOString() method returns, so I'll update documentation to reflect that version.